app/Repositories/NewOrderRepository.php

```diff
@@ -150,4 +150,12 @@ public function getDirectDebitOrderWithUnpaidAmortization(int $order_id)
             ->with('customer')
             ->first();
     }
+
+    public function changeOrderStatus(array $data)
```

Comment:
Once the validation process is properly implemented, this check will also no longer be needed as you will already have the id